pkg/api/customization/project/project_actions.go

```diff
@@ -52,36 +39,11 @@ type Handler struct {
 }
 
 func (h *Handler) Actions(actionName string, action *types.Action, apiContext *types.APIContext) error {
-	canUpdateProject := func() bool {
-		project := map[string]interface{}{
-			"id": apiContext.ID,
-		}
-
-		return apiContext.AccessControl.CanDo(v3.ProjectGroupVersionKind.Group, v3.ProjectResource.Name, "update", apiContext, project, apiContext.Schema) == nil
-	}
-
 	switch actionName {
 	case "setpodsecuritypolicytemplate":
 		return h.setPodSecurityPolicyTemplate(actionName, action, apiContext)
 	case "exportYaml":
 		return h.ExportYamlHandler(actionName, action, apiContext)
-	case "viewMonitoring":
```

Comment:
am I still going to see these actions in the API? and we are just removing the handlers for them? If yes, is it possible to hide these actions from the API completely? so user won't even see them.

Reply:
@alena1108 , is there any way to hide these actions without changing `rancher/types`?

Reply:
User can still see actions in API page: 
![image](https://user-images.githubusercontent.com/3394724/55851045-a2b61580-5b89-11e9-9abd-bcb8125fa656.png)

Reply:
@thxCode see if it can be done in formatter by removing the action from the Resource.Actions map:

https://github.com/rancher/rancher/blob/master/pkg/api/customization/cluster/formatter.go#L24